### PR TITLE
CLI rollback feature

### PR DESCRIPTION
**Pull request by silhouettes**
This feature allows you to undo your most recent release if you have shipped a broken or unintended update. It is equivalent to performing a new release with the same package contents and metadata as the release preceding your most recent one.

For performance and simplicity reasons this feature is implemented on the server side, with the implementation here being a simple POST request. In future, there is quite a lot of duplication and boilerplate in this project that can probably be refactored out (not the highest priority right now).

### CODE REVIEW

File: cli/definitions/cli.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@
     login,
     logout,
     promote,
+    rollback,
```
Comment (dtivel) [inline cli/definitions/cli.ts]: Sort

File: cli/script/command-executor.ts [hunk 2]
```diff
@@ -401,6 +401,9 @@ export function execute(command: cli.ICommand): Promise<void> {
                 case cli.CommandType.promote:
                     return promote(<cli.IPromoteCommand>command);
 
+                case cli.CommandType.rollback:
+                    return rollback(<cli.IRollbackCommand>command);
```
Comment (dtivel) [inline cli/script/command-executor.ts]: Sort

**Comment by dtivel** [comment]
Looks good.

**Comment by lostintangent** [comment]
LGTM!

**Comment by silhouettes** [comment]
Feedback:
- Specify which label was rolled back from
- Display a confirmation dialog

**Comment by shishirx34** [comment]
LGTM!

**Pull request merged**